runtime/utils.go

```diff
@@ -62,11 +63,14 @@ func PreInitMaster0(sshClient ssh.Interface, remoteHostIP string) error {
 		return fmt.Errorf("send cluster file to remote host %s failed:%v", remoteHostIP, err)
 	}
 	logger.Info("send cluster file to %s success !", remoteHostIP)
-
-	// send register login info
-	err = sshClient.Copy(remoteHostIP, common.DefaultRegistryAuthConfigDir(), common.DefaultRegistryAuthConfigDir())
-	if err != nil {
-		return fmt.Errorf("send register config to remote host %s failed:%v", remoteHostIP, err)
+	authFile := common.DefaultRegistryAuthConfigDir()
+	//if auth file exist, copy file to remote
+	if _, err := os.Stat(authFile); err == nil {
+		// send register login info
+		err = sshClient.Copy(remoteHostIP, authFile, authFile)
+		if err != nil {
+			return fmt.Errorf("send register config to remote host %s failed:%v", remoteHostIP, err)
+		}
 	}
 	logger.Info("send register info to %s success !", remoteHostIP)
```

Comment:
what is the solution if authFile not exists? still log success?

Reply:
send registry auth info , not register info.
and the success log would still be executed, I it's not allowed